python/lsst/ts/observatory/control/remote_group.py

```diff
@@ -1207,3 +1207,47 @@ async def __aenter__(self: TRemoteGroup) -> TRemoteGroup:
 
     async def __aexit__(self, *args: typing.Any) -> None:
         await self.close()
+
+    async def show_auth_status(self) -> None:
+        """Shows the authorization status of a group"""
+
+        async def _log_auth_status(c: salobj.Remote) -> None:
+            """Helper function to log authorization status"""
+            authList = await c.evt_authList.aget()
```

Comment:
Something I just realized, this is not specifying the timeout. This means that, if the data is not available, this will block for ever. I think we need a really short timeout (e.g. `self.fast_timeout`) but also, instead of timing out, we should format the message to display something like:

```
{component_name}: No auth list information available.
```

Reply:
This is automatically solved when I apply your comments above. The `_aget_topic_samples_for_components` method already considers the timeout when getting the event.